packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -14,39 +14,62 @@ The only flows that support refresh tokens are the resource owner password flow
 
-In the case of the authorization code flow, you use the authorization server's `/authorize` endpoint to get an authorization code, specifying an `offline_access` scope. You then send this code to the `/token` endpoint to get an access token and a refresh token.
+## Get a refresh token with the code flow
 
-> **Note:**  Authorization code with PKCE requests don't return refresh tokens if they are sent from SPAs or other browser-based apps. Instead, you can [silently refresh tokens](#get-a-new-access-token-id-token-silently-for-your-spa) by making a call to the `/authorize` endpoint.
+In the case of the authorization code flow, you use the Authorization Server's `/authorize` endpoint to get an authorization code, specifying an `offline_access` scope. You then use the `authorization_code` grant with this code in a request to the `/token` endpoint to get an access token and a refresh token.
 
 See [Obtain an authorization grant from a User](/docs/reference/api/oidc/#authorize) and [Implementing the authorization code flow](/docs/guides/implement-auth-code/) for more information on the `/authorize` endpoint and the authorization code flow.
 
-#### Example request for an authorization code and refresh token
+### Example request for an authorization code and refresh token
 
-The following is an example request to the `/authorize` endpoint for an authorization code and includes the `offline_access` scope.
+The following is an example request to the `/authorize` endpoint for an [authorization code](/docs/guides/implement-auth-code/overview/) and includes the `offline_access` scope.
 
 ```bash
 GET https://${yourOktaDomain}/oauth2/default/v1/authorize?client_id={clientId}
  &response_type=code
  &scope=openid%20offline_access
  &redirect_uri=http%3A%2F%2Flocalhost%3A8080
- &state=1
+ &state=8610b61u-56d1-4dsa-987c-806c3d8967j9
 ```
 
-#### Example request for an access token, ID token, and refresh token
+The following is an example request to the `/authorize` endpoint for an [authorization code with PKCE](/docs/guides/implement-auth-code-pkce/overview/) and includes the `offline_access` scope.
 
-The following is an example request to the `/token` endpoint to obtain an access token, an ID token (by including the `openid` scope), and a refresh token. The value for `code` is the code that you receive in the response from the request to the `/authorize` endpoint.
+```bash
+https://${yourOktaDomain}/oauth2/default/v1/authorize?client_id={clientId}
+&response_type=code
+&scope=openid%20offline_access
+&redirect_uri=yourApp%3A%2Fcallback
+&state=8600b31f-52d1-4dca-987c-386e3d8967e9
```

Comment:
nit: this value is very similar to the previous example `8610b61u-56d1-4dsa-987c-806c3d8967j9`. Perhaps generate a set of different UUID values and use them in the examples instead from here: https://www.uuidgenerator.net/

Reply:
done